_posts/setting-up-natty.md

```diff
@@ -0,0 +1,297 @@
+This is based entirely off of @BhargavRao's [answer](https://stackoverflow.com/c/sobotics/a/152/23) on Channels/Teams, but meant to be a complete walk-through to those who want to set it up but are not techies.
```

Comment:
A small introduction would be more helpful here. There's no need to link to the other answer, because it's a public place.